Foundation/NSStream.swift

```diff
@@ -109,24 +109,154 @@ public class Stream: NSObject {
     }
     
     /*@NSCopying */public var streamError: NSError? {
-        NSUnimplemented()
+        NSRequiresConcreteImplementation()
+    }
+    
+    public override func copy() -> AnyObject {
+        return self
+    }
+    
+    public func copy(with zone: NSZone?) -> AnyObject {
+        return self
+    }
+}
+
+internal final class _SwiftInputStream : NSInputStream, _SwiftNativeFoundationType {
+    internal typealias ImmutableType = NSInputStream
+    internal typealias MutableType = NSMutableData
```

Comment:
Mutable type as NSMutableData as a holder for now... as not sure what else to use for now